content/ember/v3/jquery-apis.md

```diff
@@ -0,0 +1,122 @@
+---
+id: jquery-apis
+title: Replace jQuery APIs
+until: '4.0.0'
+since: '3.8'
+---
+
+As jQuery will be removed from Ember by default, APIs that are coupled to jQuery have been deprecated.
```

Comment:
```suggestion
As of Ember 3.2.0, Ember no longer requires that all applications include jQuery, therefore APIs that are coupled to jQuery have been deprecated.
```

Reply:
I believe making jQuery optional was implemented for Ember.js 3.3, wasn't it? 
But there was another thing, IIRC the 3.3 release of Ember CLI was postponed (making it 3.4), which included the necessary changes required by the (first) RFC. So we referred to 3.4 as the version which made jQuery optional, as this was the least confusing option for users (who usually have Ember & CLI in lockstep). This is at least what the Guides say: https://guides.emberjs.com/release/configuring-ember/optional-features/#toc_jquery-integration

So I changed this to 3.4.0 now, hope this is fine?